message.go

```diff
@@ -149,6 +149,33 @@ var coapOptionDefs = map[OptionID]optionDef{
 	NoResponse:    optionDef{valueFormat: valueUint, minLen: 0, maxLen: 1},
 }
 
+var optionNames = [...]string{
+	IfMatch:       "IfMatch",
+	URIHost:       "URIHost",
+	ETag:          "ETag",
+	IfNoneMatch:   "IfNoneMatch",
+	Observe:       "Observe",
+	URIPort:       "URIPort",
+	LocationPath:  "LocationPath",
+	URIPath:       "URIPath",
+	ContentFormat: "ContentFormat",
+	MaxAge:        "MaxAge",
+	URIQuery:      "URIQuery",
+	Accept:        "Accept",
+	LocationQuery: "LocationQuery",
+	Block2:        "Block2",
+	Block1:        "Block1",
+	Size2:         "Size2",
+	ProxyURI:      "ProxyURI",
+	ProxyScheme:   "ProxyScheme",
+	Size1:         "Size1",
+	NoResponse:    "NoResponse",
+}
+
+func (o OptionID) String() string {
```

Comment:
Pls could you change function to similar like https://github.com/go-ocf/go-coap/blob/master/codes/code_string.go ? Because we want to print unknown option.

Reply:
Thank you for your review. I have made the change as you requested. 👍